wlutil/config.py

```diff
@@ -411,6 +419,20 @@ def applyBase(self, baseCfg):
             self.cfg['run'] = getOpt('wlutil-dir') / 'null_run.sh'
             self.cfg['runSpec'] = RunSpec(script=self.cfg['run'])
 
+        # To avoid needlessly recompiling kernels, we check if the child has
+        # the exact same binary-related configuration.
+        self.cfg['use-parent-bin'] = True
```

Comment:
Setting `"use-parent-bin": false` should unconditionally force a kernel rebuild?

Reply:
not unconditionally. "use-parent-bin" = False means that if the child binary isn't up to date, the whole kernel should be rebuilt (if nothing has changed, it won't be rebuilt). "use-parent-bin" = True means that if the child binary isn't up to date, don't compile a whole new binary, just copy your parent's.

In other words, that flag only affects what it means to generate a binary for a workload (copy from the parent vs compile from scratch).